libpod/runtime.go

```diff
@@ -263,7 +263,14 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 	configPath := ConfigPath
 	foundConfig := true
 	if rootless.IsRootless() {
-		configPath = filepath.Join(os.Getenv("HOME"), ".config/containers/libpod.conf")
+		home := os.Getenv("HOME")
+		if runtime.config.SignaturePolicyPath == "" {
+			newPath := filepath.Join(home, ".config/containers/policy.json")
+			if _, err := os.Stat(newPath); err == nil {
+				runtime.config.SignaturePolicyPath = newPath
+			}
+		}
```

Comment:
should we error out here if runtime.config.SignaturePolicyPath == "" by the end of this if statement?  I think yes, but could be convinced otherwise.